src/uwtools/config/jinja2.py

```diff
@@ -157,21 +157,22 @@ def dereference(val: _YAMLVal, context: dict, local: Optional[dict] = None) -> _
 
 
 def render(
-    input_file: OptionalPath,
-    output_file: OptionalPath,
-    values_file: DefinitePath,
+    values: Union[dict, DefinitePath],
     values_format: Optional[str] = None,
+    input_file: OptionalPath = None,
+    output_file: OptionalPath = None,
     overrides: Optional[Dict[str, str]] = None,
     values_needed: bool = False,
     dry_run: bool = False,
 ) -> bool:
```

Comment:
A Jinja2 template can now be rendered via values in an in-memory `dict` as well as via the existing file/stream-based sources.